syncd/Syncd.cpp

```diff
@@ -203,9 +203,10 @@ void Syncd::performStartupLogic()
 {
     SWSS_LOG_ENTER();
 
-    // ignore warm logic here if syncd starts in Mellanox fastfast boot mode
+    // ignore warm logic here if syncd starts in fastboot or in Mellanox fastfast boot mode
 
-    if (m_isWarmStart && (m_commandLineOptions->m_startType != SAI_START_TYPE_FASTFAST_BOOT))
+    if (m_isAdvancedStart && (m_commandLineOptions->m_startType != SAI_START_TYPE_FASTFAST_BOOT) &&
+            (m_commandLineOptions->m_startType != SAI_START_TYPE_FAST_BOOT))
```

Comment:
Can it be written with a simpler condition?

```suggestion
    if (m_isAdvancedStart && m_commandLineOptions->m_startType == SAI_START_TYPE_WARM_BOOT)
```